testsuite/integration/smoke/pom.xml

```diff
@@ -640,6 +640,45 @@
             </build>
         </profile>
 
+        <profile>
```

Comment:
Can you instead add a copy global properties and use those in the top level build/plugins section. Then just change the value of the props in this profile instead of redeclaring all the plugin stuff.

Reply:
I'm not sure I understand you well, but I redistributed and minimized the definitions.